contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -185,6 +365,93 @@ NXmpes(NXobject):
           exists: recommended
           doc: |
             This is the momentum axis to be used for data plotting.
+      energy_referencing(NXcalibration):
+        exists: optional
+        doc:
+        - |
+          For energy referencing, the measured energies are corrected for the charging potential
+          (i.e., the electrical potential of the surface region of an insulating sample, caused by
+          irradiation) such that those energies correspond to a sample with no surface charge.
+          Usually, the energy axis is adjusted by shifting all energies uniformally until one
+          well-defined emission line peak (or the Fermi edge) is located at a known _correct_ energy.
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 12.74 ff.
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.74
+        applied(NX_BOOLEAN):
+          doc: |
+            Have the energy axes been adjusted (e.g., in cases where samples are not
+            conductive)?
+        level(NXelectron_level):
+          exists: recommended
+          doc: |
+            Electronic core or valence level that was used for the calibration.
+        reference_peak:
+          doc: |
+            Reference peak that was used for the calibration.
+            
+            For example: adventitious carbon | C-C | metallic Au | elemental Si | Fermi edge
+        binding_energy(NX_FLOAT):
+          exists: recommended
+          doc:
+          - |
+            The binding energy (in units of eV) that the specified emission line appeared at,
+            after adjusting the binding energy scale.          
+          - |
+            xref:
+              spec: ISO 18115-1:2023
+              term: 12.16_ ff.
+              url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+        offset(NX_FLOAT):
+          exists: recommended
+          doc: |
+            Offset between measured binding energy and calibrated binding energy of the
+            emission line.
+        calibrated_axis(NX_FLOAT):
+          exists: recommended
+          doc: |
+            This is the calibrated energy axis to be used for data plotting.
+            
+            This should link to /entry/data/energy.
+      transmission_correction(NXcalibration):
+        exists: optional
+        doc: |
+          In the transmission correction, each intensity measurement for electrons of a given
+          kinetic energy is multiplied by the corresponding value in the relative_intensity
+          field of the transmission_function. This calibration procedure is used to account for
+          the different tranmsission efficiencies when using different lens modes.
+        applied(NX_BOOLEAN):
+          doc: |
+            Has an intensity calibration been applied?
+            
+            That is, has the transmission function of the analyser been taken into account?
```

Comment:
I am wondering whether it makes sense to not remove all of these "applied" fields. At the end of the day, you just don't provide the group if you did _not_ apply the correction, no?

Reply:
Please do so, and resolve

Reply:
Done